scheduler.go

```diff
@@ -570,3 +571,15 @@ func (s *Scheduler) scheduleAllJobs() {
 		s.scheduleNextRun(j)
 	}
 }
+
+func handleOptions(options ...interface{}) {
+	for _, option := range options {
+		switch option.(type) {
+		case Logger:
+			// Note: The logger will only be initilized if the Logger
+			// interface is conformed to. The logs will not be printed
+			// in this case
+			initLogger(option.(Logger))
```

Comment:
shouldn't the logger be attached to a scheduler instance? considering a log struct may hold state itself as well

Reply:
Makes sense!
Instead of making the `logger` global, I'll make it a member of `Scheduler`. In which case, the `handleOptions` function can also become a method of `Scheduler` right?

Reply:
a good approach to optional parameters that I use [looks like this](https://stackoverflow.com/a/26326418/1670308)

Reply:
:100: This is really cool. Will implement this. Thanks!